v0/sources/iterable/mapping.json

```diff
@@ -0,0 +1,5 @@
+{
+  "email": ["context.traits.email"],
+  "userId": ["userId", "context.traits.userId"],
+  "dataFields": "properties"
```

Comment:
As we are already mapping `email` and `userId` this will get duplicated in properties

Reply:
For events like `Hosted unsubscribe click`, Iterable sends `userId` as "1", which is different from 'email'.

Reply:
`userId` is an unique identifier which is different from `email`

Reply:
Got it, removed `email` from `properties`